From e721a5de5ecd236aacaf986f98332b1ac11277d2 Mon Sep 17 00:00:00 2001
From: zachary <zhangzg@marvell.com>
Date: Tue, 28 Mar 2017 17:48:36 +0800
Subject: [PATCH 0707/1200] defconfig: a3700: enable pca953x

- There is PCA953x io-expander on db board.
- This patch enables io-expander as gpio function.

Change-Id: Icac9cfe1f18a697715c7d69516c04231d377a9e7
Signed-off-by: zachary <zhangzg@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38085
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hua Jing <jinghua@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/55108
Tested-by: Hua Jing <jinghua@marvell.com>
---
 configs/mvebu_db-88f3720_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/mvebu_db-88f3720_defconfig b/configs/mvebu_db-88f3720_defconfig
index 96017faac7..2d0a6ec192 100644
--- a/configs/mvebu_db-88f3720_defconfig
+++ b/configs/mvebu_db-88f3720_defconfig
@@ -37,6 +37,7 @@ CONFIG_AHCI_MVEBU=y
 CONFIG_BLOCK_CACHE=y
 CONFIG_DM_GPIO=y
 # CONFIG_MVEBU_GPIO is not set
+CONFIG_DM_PCA953X=y
 CONFIG_DM_I2C=y
 CONFIG_MISC=y
 CONFIG_DM_MMC=y
-- 
2.22.0

